docs/get-started/lotus/installation.md

```diff
@@ -363,23 +397,63 @@ We recommend waiting until the syncing process has completed, which should be re
 lotus sync wait
 ```
 
+While waiting the syncing process, you can also check  status of the syncing process.
```

Comment:
```suggestion
While you wait, you can check the status of the syncing process:
```

Simplify for clarity.

Reply:
This sections is a duplicate of line 427. Let's keep one, not both. I prefer the placement in line 427 but up to you.

Reply:
Make sence, I will remove keep line 427 since `lotus sync wait` will also print out the status.